src/cmd/new.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"github.com/go-git/go-git/v5"
+	"github.com/nasz-elektryk/spito/checker"
+	"github.com/nasz-elektryk/spito/cmd/cmdApi"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+)
+
+var newRulesetCommand = &cobra.Command{
+	Use:   "new {ruleset_name}",
+	Short: "Create new ruleset",
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		rulesetName := strings.ReplaceAll(args[0], " ", "")
+		gitCommand := exec.Command("git", "config", "--global", "--list")
```

Comment:
I think it is better idea to use library for git (especially that checker module already imports library for git)